docs/src/development/transfer-dedicated.md

```diff
@@ -122,19 +157,24 @@ Which should give a JSON output containing something like this:
 }
 ```
 
-The part we want is the host, user, password, and the "path", which is the database name.
-Ignore the rest.
+From which you need the following values:
+
+- `username`
+- `password` (not needed if empty)
+- `path`: the database name (`<DATABASE_NAME>` in the example command below)
+
+You also need the [SSH username and host](#get-the-ssh-username-and-host) for your environment.
 
 Now, in a separate terminal log in to the development instance using `platform ssh`.
 Run the same `echo` command as above to get the credentials for the database on the development instance.
-(The JSON will be slightly different but again we're only interested in the user, password, host, and "path"/database name).
+Again, we're only interested in the username, password, host, and "path"/database name.
 
 With the credentials from both databases,
-we can construct a command that will export data from the development server
+we can construct a command that exports data from the development server
 and write it directly to the Dedicated cluster's server.
 
 ```bash
-mysqldump -u <dev_user> -p<dev_password> -h <dev_host> <dev_dbname> --single-transaction | ssh -C <USERNAME>@<CLUSTER_NAME>.ent.platform.sh 'mysql -u <prod_user> -p<prod_password> -h <prod_host> <prod_dbname>'
+mysqldump -u <DEV_USER> -p <DEV_PASSWORD> -h <DEV_HOST> <DEV_DATABASE_NAME> --single-transaction | ssh -C <SSH_USERNAME>@<host> 'mysql -u <PROD_USER> -p <PROD_PASSWORD> -h <PROD_HOST> <PROD_DATABASE_NAME>'
```

Comment:
:clipboard: 
```suggestion
mysqldump -u <DEV_USER> -p <DEV_PASSWORD> -h <DEV_HOST> <DEV_DATABASE_NAME> --single-transaction | ssh -C <SSH_USERNAME>@<HOST> 'mysql -u <PROD_USER> -p <PROD_PASSWORD> -h <PROD_HOST> <PROD_DATABASE_NAME>'
```

That's a lot of `host`s. Maybe `SSH_HOST` for the one? And match that also above? Not sure if it's necessary :shrug: